beaker-vue/src/components/notebook/BeakerNotebookToolbar.vue

```diff
@@ -108,6 +108,68 @@
             </slot>
         </template>
     </Toolbar>
+    <Dialog
```

Comment:
I think the dialog here is fine in itself, but I'm not sure if we can be confident that we will only want to show it in situations in which the beaker notebook toolbar is shown, e.g. the chat view.

I think we will be adding an option to export from chat soon, but are not planning on putting the full toolbar with adding/removing cells, etc.

Probably makes sense to set it up as [a dynamic dialog](https://primevue.org/dynamicdialog/).
There already exists a `<DynamicDialog/>` tag in BaseInterface.vue, so I think you can just import the new component and use useDialog to display the dialog.